app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -70,17 +60,19 @@ def update
         )
 
         if result.success?
-          render(
-            json: ::V1::SubscriptionSerializer.new(
-              result.subscription,
-              root_name: 'subscription',
-            ),
-          )
+          render_subscription(result.subscription)
         else
           render_error_response(result)
         end
       end
 
+      def show
+        subscription = current_organization.subscriptions.find_by(external_id: params[:external_id])
```

Comment:
`external_id` could belongs to multiple subscription. Should we return only the active one? (and maybe have an ability to retrieve a terminated?)